### PR TITLE
morph/client: Ignore error if a transaction already exists

### PR DESCRIPTION
**Pull request by fyrchik**
Signed-off-by: Evgenii Stratonikov <evgeniy@morphbits.ru>

### CODE REVIEW

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing.

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing.

**Comment by sami-nspcc** [comment]
I am running integration tests

**Comment by sami-nspcc** [comment]
I am running integration tests

**Comment by sami-nspcc** [comment]
I have recieved a notification of a new pull request. I am starting the build of images and binaries for further testing.

**Comment by sami-nspcc** [comment]
Test run is finished. Please download the tarball from `st*.t5.fs.neo.org`:  FszwwAx3VznDysQGxBUMRj1zj3zp8VcohorVDnunrviV/BQQBdewRsHgoD1suUwpriQm5L7wwfEpTSESVWr27XxQy Untar and use `allure open` to watch the report

**Review by acid-ant — APPROVED** [review]

File: pkg/morph/client/notary.go [hunk 1]
```diff
@@ -175,7 +176,18 @@ func (c *Client) DepositNotary(amount fixedn.Fixed8, delta uint32) (res util.Uin
...
+		if !errors.Is(err, neorpc.ErrAlreadyExists) {
+			return util.Uint256{}, fmt.Errorf("can't make notary deposit: %w", err)
+		}
+
+		// Transaction is already in mempool waiting to be processed.
+		// This is an expected situation if we restart the service.
+		c.logger.Debug("notary deposit has already been made",
+			zap.Int64("amount", int64(amount)),
+			zap.Int64("expire_at", till),
+			zap.Uint32("vub", vub),
```
Comment (carpawell) [inline pkg/morph/client/notary.go:188]: suggest _one of_:

1. "vub" -> "valid until block" if we try to be more friendly for a user
2. "expire_at" -> "till" if we want to save the original `neo-go` context

**Review by carpawell — APPROVED** [review]

File: pkg/morph/client/notary.go (hunk 1 above)
Comment (fyrchik) [reply pkg/morph/client/notary.go:188]: I have copied it from the line below, let's be consistent here.

**Pull request merged**